zigpy/zcl/__init__.py

```diff
@@ -682,10 +682,10 @@ def write_attributes_undivided(
         args = self._write_attr_records(attributes)
         return self._write_attributes_undivided(args, manufacturer=manufacturer)
 
-    def bind(self):
+    async def bind(self):
         return self._endpoint.device.zdo.bind(cluster=self)
```

Comment:
You will need to `await` the underlying function call as well:

```suggestion
        return await self._endpoint.device.zdo.bind(cluster=self)
```

Reply:
Thanks, done in 8f64a19eaf061d0b02604735ecba4e29c6365769.